tests/modules/bson.rs

```diff
@@ -131,12 +131,12 @@ fn from_impls() {
         Bson::Document(doc! {"a": "b"})
     );
 
-    let db_pointer = Bson::from_extended_document(doc! {
+    let db_pointer = Bson::from(json!({
```

Comment:
Interestingly, now that `from_extended_document` is private, the only ways to create a `DbPointer` are through JSON or raw bytes. Since this is just a test, I picked the easier one.